packages/world/src/Constants.sol

```diff
@@ -6,7 +6,7 @@ uint32 constant MAX_PICKUP_RADIUS = 6;
 uint32 constant MAX_RESPAWN_HALF_WIDTH = 10;
 
 uint16 constant MAX_PLAYER_JUMPS = 3;
-uint16 constant MAX_PLAYER_GLIDES = 5;
+uint16 constant MAX_PLAYER_GLIDES = 10;
```

Comment:
you were seeing reverts from glides?

Reply:
yeah, I think mostly due to path being recomputed + jumping

Reply:
I see. I think it'd be good to review the recompute move logic btw. I wrote it wayy back May 2023, but someone else reviewing it will help a lot, and we prob need to update it